starter-code/MyLinkedList.java

```diff
@@ -13,50 +13,118 @@ public int getSize() {
 
 	public boolean remove(int index) {
 		//to-do
+
+		/*
+		head
+		 */
+		if (index == 0) {
+
+			head = head.getNext();
+
+		/*
+		tail
+		 */
+		} else if (index +1 == size) {
+
+			getNodeAt(index - 1).setNext(null);
+
+		/*
+		previous node
+		*/
+		} else {
+
+			Node<T> previousNode = getNodeAt(index -1);
+			Node<T> nextNode = previousNode.getNext().getNext();
+
+			previousNode.setNext(nextNode);
+		}
 	}
 
 	public T get(int index) {
 		//to-do
+		Node<T> currentNode = getNodeAt(index);
+		return currentNode.getData();
+	}
+
+	public Node<T> getNodeAt(int index) {
+
+		Node<T> currentNode = head;
+
+		for (int i = 0; i < index; i++) {
+
+			currentNode = currentNode.getNext();
```

Comment:
Looks like you forgot to return the node at the end of this method, and you have some mismatched brackets